DeepSpeech.ipynb

```diff
@@ -1270,6 +1273,34 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "For reporting we also need a standard way to do time measurements."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# This function will toggle a stopwatch\n",
+    "# The first call starts it, second call stops it, third call continues it etc.\n",
```

Comment:
I see what you mean now, but I think this comment is misleading. This isn't really describing what it does, so much as how you'd want to use it. I'd prefer a more descriptive comment that says what it does and an annotated example of how to use it as it took me a couple of minutes to wrap my head around it.